concise-evidence.cddl

```diff
@@ -21,6 +21,7 @@ ev-coswid-triple-record = [
 
 ev-coswid-evidence-map = non-empty< { 
     ? ce.coswid-tag-ids => [ + concise-swid-tag-id ], 
-    ? ce.coswid-evidence => evidence-entry 
+    ? ce.coswid-evidence => evidence-entry,
+    ? ce.authorized-by => [ + $crypto-key-type-choice ] ; see comid schema
 } > 
 concise-swid-tag-id = tag-id
```

Comment:
This was removed in a different PR. Not sure why it's not showing up in the diffs.

Reply:
Could you please show what is supposed to be here via the suggestion feature?

Reply:
Not sure how to use suggestion feature. concise-swid-tag-id = tag-id was changed by Thomas in a different PR. Then realized it wasn't needed since it also existed in comid-extensions.cddl. I removed it in a followup PR. I resolved the conflicts in this PR due to the other updates. Moving ahead with committing this PR.